app/views/two_factor_authentication/shared/max_login_attempts_reached.html.erb

```diff
@@ -0,0 +1,18 @@
+<% lockout_time_in_words = decorated_user.lockout_time_remaining_in_words %>
```

Comment:
I don't think a helper method is needed, but I do think that using that veryyyy long `decorated_user.lockout_time_remaining_in_words` directly in the markup would be fine by me!